### DISCUSSION TITLE
JS vs JAR

### DISCUSSION THREAD

**Discussion by iacsha** [odoodo/Mirth-Migrator#10] [Q&A] [answered]
Java newb here...

Instructions say to link the Resources to web\MirthMigrator\jar
In the download there is only the JS subfolder in MirthMigrator

Do I change the folder name to JAR, extensions, run a magical config script?

Thanks

**Comment by jonbartels** [comment, marked as answer]
https://github.com/odoodo/Mirth-Migrator/issues/9

**Reply by iacsha** [reply]
D'oh. Thanks Jon.
